infra/Pos/Network/CLI.hs

```diff
@@ -104,9 +103,11 @@ defaultTopology = Y.TopologyBehindNAT defaultDnsDomains
 
 -- | The default DNS domains used for relay discovery
 --
-defaultDnsDomains :: DnsDomains
 -- TODO: Give this a proper value
-defaultDnsDomains = DnsDomains [["todo.defaultDnsDomain.com"]]
+defaultDnsDomains :: DnsDomains DNS.Domain
+defaultDnsDomains = DnsDomains [
+      [NodeAddrDNS "todo.defaultDnsDomain.com" Nothing]
```

Comment:
Ultimately this will need to come from the constants.yaml file which is available at compile time (due to devious hackery). Doesn't need to be fixed in this PR though.